src/Microsoft.ML.FastTree/TreeEnsembleFeaturizer.cs

```diff
@@ -488,7 +422,7 @@ private void GetTreeSlotNames(int col, ref VBuffer<ReadOnlyMemory<char>> dst)
             dst = editor.Commit();
         }
 
-        private void GetLeafSlotNames(int col, ref VBuffer<ReadOnlyMemory<char>> dst)
+        private void GetLeafSlotNames(ref VBuffer<ReadOnlyMemory<char>> dst)
```

Comment:
_(off topic for the current PR)_

How deep are normal trees? Could we name the slots for paths/leaves as "Tree033Leaf021-MyFeatABC_MyFeatXYZ_MyFeatIOU..." to better help users understand the output of the feature importance? The downsize is the name needs to be unique, and will be long as it notes all the input features the leaf node uses.

The current slot naming is a bit useless (though much better than nothing).
Currently the slot names look like:
```C#
LeavesMainLabel.Tree033Leaf021	65.73584
LeavesMainLabel.Tree033Leaf023	-42.43543
LeavesMainLabel.Tree033Leaf019	-40.72021
LeavesMainLabel.Tree057Leaf020	-37.54552
LeavesMainLabel.Tree079Leaf007	-36.29255
LeavesMainLabel.Tree055Leaf019	-34.78884
LeavesMainLabel.Tree075Leaf009	34.58635
LeavesMainLabel.Tree020Leaf020	33.72996
LeavesMainLabel.Tree047Leaf022	31.86535
LeavesMainLabel.Tree074Leaf008	31.86535
LeavesMainLabel.Tree066Leaf008	31.74181
LeavesMainLabel.Tree040Leaf019	30.9242
```